docs/work/2026-06-06-kernel-backlog-memory-roadmap/D19-filesystem-doctor-design.md

```
@@ -0,0 +1,261 @@
+  │ D19-T2 doctor cmd │   │ D19-T3 broker default-on    │
+  │ lib/commands/     │   │ gate                         │
+  │   doctor.js       │   │ lib/kernel/broker.js         │
+  │ (+ helper reuse   │   │ (initialize() hook only)     │
+  │  of T1 + broker   │   └────────────────────────────┘
+  │  config builder)  │
+  └───────────────────┘
+Wave 3 (optional, after both): D19-T4 docs/AGENTS surface note — non-code, sync-commands if needed.
+```
+
+### File ownership (no collisions)
+| Task | OWNS (writes) | READS only |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)